bootstrap/sql/org.postgresql.Driver/v004__create_db_connection_info.sql

```diff
@@ -100,3 +100,8 @@ CREATE TABLE IF NOT EXISTS openmetadata_settings (
     UNIQUE(configType)
 );
 
+DELETE FROM entity_extension
+WHERE jsonSchema IN ('tableProfile', 'columnTest', 'tableTest');
+
+delete from ingestion_pipeline_entity 
+where json_extract_path_text("json", 'pipelineType') = 'profiler'
```

Comment:
nit - missing a semicolon here. IDK if it can impact flyway if we need to add more statement below